tests/functional/services/api/registries/__init__.py

```diff
@@ -3,8 +3,8 @@
 
 def get_registry_info():
     return {
-        'user': os.environ.get('ANCHORE_TEST_DOCKER_REGISTRY_USER', 'anchore'),
```

Comment:
why remove the defaults?

Reply:
[ditto](https://github.com/anchore/anchore-engine/pull/679#discussion_r506653510)